packages/asc-ui/src/components/Footer/FooterHeading/FooterHeading.tsx

```diff
@@ -1,12 +1,17 @@
-import styled from '@datapunt/asc-core'
-import Heading from '../../Heading'
-import { color } from '../../../utils'
-
-const FooterHeading = styled(Heading)`
-  margin-top: 0;
-  margin-bottom: 8px;
-  color: ${color('tint', 'level1')};
-  display: flex;
-`
+import React from 'react'
+import FooterHeadingStyle, { Props } from './FooterHeadingStyle'
+
+export { Props }
+
+const FooterHeading: React.FC<
+  Props & React.HTMLAttributes<HTMLHeadingElement>
+> = ({ children, ...otherProps }) => {
+  return <FooterHeadingStyle {...otherProps}>{children}</FooterHeadingStyle>
+}
+
+FooterHeading.defaultProps = {
+  $as: 'h3',
+  styleAs: 'h3',
```

Comment:
remove styleAs, this is optional and doesn't do anything if not defined :)

Reply:
removed